server/container/guice/cassandra-rabbitmq-ldap-guice/src/test/resources/ldif-files/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM dinkel/openldap:latest
+
+COPY populate.ldif /etc/ldap/prepopulate/prepop.ldif
```

Comment:
Given that we import LdapTestExtension from cassandra-ldap-guice mvn module, do we really need to duplicate this LDIF and docker file?